ui/components/src/EChart.tsx

```diff
@@ -11,49 +11,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import React, { useState, useLayoutEffect } from 'react';
+import React, { useCallback, useEffect, useLayoutEffect, useRef } from 'react';
 import debounce from 'lodash/debounce';
 import { ECharts, EChartsCoreOption, init } from 'echarts/core';
 import { Box, SxProps, Theme } from '@mui/material';
+import { isEqual } from 'lodash-es';
 
+// https://echarts.apache.org/en/api.html#events
```

Comment:
Removed these comments since they are duplicated verbatim from [this link](https://echarts.apache.org/en/api.html#events)